messaging/content-templates/create-location-template/create-location-template.curl

```diff
@@ -0,0 +1,17 @@
+curl -X POST 'https://content.twilio.com/v1/Content' \
+-H 'Content-Type: application/json' \
+-u $TWILIO_ACCOUNT_SID:$TWILIO_AUTH_TOKEN \
+-d '{
+    "friendly_name": "Owl Air location template",
+    "language": "en",
+    "types": {
+        "twilio/text": {
+            "body": "Owl Air: Time to board, SFO is located at San Francisco International Airport, P.O. Box 8097, San Francisco, CA 94128 "
+        },
+        "twilio/location": {
```

Comment:
I think I'm missing this -- what is the templated part here? Is the location sent separately from the body text?